cmd/kg/main.go

```diff
@@ -92,6 +92,7 @@ func Main() error {
 	local := flag.Bool("local", true, "Should Kilo manage routes within a location?")
 	logLevel := flag.String("log-level", logLevelInfo, fmt.Sprintf("Log level to use. Possible values: %s", availableLogLevels))
 	master := flag.String("master", "", "The address of the Kubernetes API server (overrides any value in kubeconfig).")
+	topologyLabel := flag.String("topology-label", k8s.RegionLabelKey, "Kubernetes node label used to group nodes into logical locations.")
```

Comment:
It just occurred to me, should we be clear that this is the label _key_? Or is that not necessary?

Maybe we can look at what K8s calls this now on their well known labels page https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/
They make no mention of "key" only "label" and "value". So maybe this is fine as it. WDYT?

Reply:
I think "label" is pretty clear :]